crates/sui-config/src/node.rs

```diff
@@ -334,7 +334,7 @@ impl Default for AuthorityStorePruningConfig {
         Self {
             num_latest_epoch_dbs_to_retain: usize::MAX,
             epoch_db_pruning_period_secs: u64::MAX,
-            num_epochs_to_retain: 0,
+            num_epochs_to_retain: 1,
```

Comment:
Just to be extra safe, maybe we can set it to 2 or 3, similar to `num_latest_epoch_dbs_to_retain`?